programs/sponsors/sponsors.md

```diff
@@ -0,0 +1,61 @@
+# Sponsors
+
+JSON Schema uses [OpenCollective](https://opencollective.com/json-schema) and [GitHub Sponsors](https://github.com/sponsors) to gather money. This money is used to cover project expenses in a transparent way ([See the OpenCollective documentation](https://opencollective.com/how-it-works)), and your donation will help to sustain and grow the Project.
```

Comment:
I might consider mentioning that we (I think?) plan to move any funds collected via GH Sponsors to OC, and therefore they too will be transparently spent.

It might even be worth having a section explicitly called "what's preferred, contributing via OC or GH Sponsors" or whatever, where we answer whether we have a preference on where someone contributes. (I don't recall if there's any financial benefit to one or the other -- presumably not and therefore we don't actually care which).